### ISSUE TITLE
Erlang/OTP support

### ISSUE THREAD

**Issue by Pouriya-Jahanbakhsh** [dgiagio/warp#16]
I like to help for supporting Erlang. What do we need to make it possible?

**Comment by dgiagio** [comment]
Hi, could you please submit a PR? I would be happy to review and accept it.
Thanks.

**Comment by Pouriya-Jahanbakhsh** [comment]
Can we have an `examples` directory?

**Comment by dgiagio** [comment]
Yes, that's a good idea. Please avoid submitting binary files, though. Text files, scripts, etc. should be fine.
